Library_Engine/Query/Libraries.cs

```diff
@@ -228,7 +228,7 @@ private static void AddToPathDictionary(string path, string dictionaryPath)
         /**** Private Fields                            ****/
         /***************************************************/
 
-        private static readonly string m_sourceFolder = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), @"BHoM\DataSets");
+        private static readonly string m_sourceFolder = @"C:\ProgramData\BHoM\DataSets";
```

Comment:
```suggestion
        private static readonly string m_sourceFolder = @"C:\ProgramData\BHoM\Datasets";
```

Hey @IsakNaslundBh - can we make this `Datasets` here? Where else would this need a change?

I was just posting on the default post-build events for toolkits - as well as specific for BHoM.csproj which can assist in making directories so good time to confirm

Reply:
Happy to leave as is if painful as part of this PR as discussed with regards to other tweaks

Reply:
update done. Agree with this change, as long, as you mentioned, we track it through in all postbuilds :)